railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -205,6 +211,7 @@ def create_root_files
         build(:configru)
         build(:gitignore)   unless options[:skip_git]
         build(:gemfile)     unless options[:skip_gemfile]
+        build(:version_control)
```

Comment:
Since other developers can create custom AppBuilder templates by overriding rails AppBuilder methods, adding `options[:skip_git]` here would add more complexity if someone wants to use other `version_control` system in their templates :)